libs/solid/core/src/lib/components/audio-toolbar/audio-toolbar.component.html

```diff
@@ -1,19 +1,32 @@
-<div class="audiotoolbar">
-  <button *ngIf="description" mat-icon-button (click)="toggleDescription()">
-    <mat-icon>description</mat-icon>
-  </button>
-  <button mat-icon-button (click)="onPlayPauseClick()">
+<div class="audioToolbar">
+  <button (click)="onPlayPauseClick()" mat-mini-fab color="accent">
     <mat-icon>{{ Playing ? 'pause' : 'play_arrow' }}</mat-icon>
+    <!-- <ng-template
```

Comment:
Eigentlich sollte vor einem finalen Merge nichts mehr auskommentiert sein, zumindest nicht ganze Teile.
Wenn Du Teile später noch benutzen möchtest, dann kannst Du sie aus diesem Commit holen und z.B. in einen neuen Branch schieben.
Bitte vor dem Merge noch alles entfernen, was nicht konkret gebraucht wird.

Reply:
Vielleicht kannst Du auch gleich einen neuen Branch anlegen, an dem Du später weitermachen kannst.